react_shiftme/README.md

```diff
@@ -0,0 +1,68 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
Are necessary 3 files of README?

Reply:
Yes, At the moment we will keep the 3 files 1 generated, 1 frontend and 1 backend.